frontend/src/app/general/auth/auth/auth.component.ts

```diff
@@ -41,8 +41,6 @@ export class AuthComponent implements OnInit {
   }
 
   webSSO() {
-    this.authService.getRedirectURL().subscribe((res) => {
-      window.location.href = res.auth_url;
-    });
+    this.authService.webSSO();
   }
```

Comment:
I assume this can be replaces by calling `this.authService.webSSO()` directly.

Reply:
I wanted to leave it in, since `webSSO()` is invoked from the html template as well.

Reply:
I'd make it public then and replace it in the HTML template as well.

Reply:
I feel it would be better to use the component as the abstraction level, though. Referencing services directly is a first  step towards the creation of [train wrecks](https://wiki.c2.com/?TrainWreck). It always starts with one...